### ISSUE TITLE
Mecab help message is not shown when mecab is not installed

### ISSUE THREAD

**Issue by hiroshinoji** [mynlp/jigg#79]

**Comment by hiroshinoji** [comment]
Currently, the annotators for external softwares do not output the error message indicating how to install the software, which was outputted in the previous version. For example, the current CabochaAnnotator does not output the desirable error.

```
$java ... jigg.pipeline.Pipeline -annotators "ssplit,kuromoji,cabocha"
...
ERROR: Failed to start cabocha.
  cmd: cabocha -f1 -I1
output:
bash: cabocha: command not found
```

I found the commit below changed the manner of handling launching errors. By wrapping command(s) with `bash -c`, which I added following [this discussion](https://stackoverflow.com/questions/12268550/java-processbuilder-to-start-execute-multiple-commands-sequentially-in-linux), Jigg seems to be not able to judge whether the started process is dead (cannot be executed) or not.

https://github.com/mynlp/jigg/commit/8d6a9aea54fde74e7b8b6685d06ed8adce7c9fba#diff-324790cca0ab5209ef07d28d1946a514
